### PR TITLE
GO-3392 Clean up unused imports

### PR DESCRIPTION
**Pull request by fat-fellow**
---

- [ ] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: core/block/editor.go [hunk 1]
```diff
@@ -534,8 +534,6 @@ func (s *Service) UploadFile(ctx context.Context, spaceId string, req FileUpload
 	upl.SetAdditionalDetails(req.Details)
 	if req.Type != model.BlockContentFile_None {
 		upl.SetType(req.Type)
-	} else {
-		upl.AutoType(true)
```
Comment (deff7) [inline core/block/editor.go:538]: why did you remove this?

File: core/block/editor/dataview/dataview.go [hunk 2]
```diff
@@ -434,83 +430,6 @@ func getDataviewBlock(s *state.State, id string) (dataview.Block, error) {
...
-	position int
-	entry    *types.Struct
-}
-
-type recordsInsertedAtPosition struct {
-	position int
-	entries  []*types.Struct
-}
-
-func calculateEntriesDiff(a, b []database.Record) (updated []*types.Struct, removed []string, insertedGroupedByPosition []recordsInsertedAtPosition) {
```
Comment (deff7) [inline core/block/editor/dataview/dataview.go:455]: I didn't see this code before 🤣

File: core/block/editor/file/file.go [hunk 3]
```diff
@@ -561,7 +561,6 @@ func (dp *dropFilesProcess) addFile(f *dropFileInfo) {
 	upl := dp.fileUploaderFactory.NewUploader(dp.spaceID, objectorigin.DragAndDrop())
 	res := upl.
 		SetName(f.name).
-		AutoType(true).
```
Comment (deff7) [inline core/block/editor/file/file.go:564]: another questionable change

File: core/files/fileuploader/uploader.go [hunk 4]
```diff
@@ -362,11 +360,6 @@ func (u *uploader) setLastModifiedDate(path string) {
 	}
 }
 
-func (u *uploader) AutoType(enable bool) Uploader {
-	u.typeDetect = enable
```
Comment (deff7) [inline core/files/fileuploader/uploader.go:366]: why 😿

Comment (fat-fellow) [reply core/files/fileuploader/uploader.go:366]: Nobody did anything about this flag, it is misleading :(

**Review by deff7 — APPROVED** [review]
